Tests/SonarQube.Bootstrapper.Tests/BuildAgentUpdaterTests.cs

```diff
@@ -64,7 +64,7 @@ public void Updater_CheckDownloadUrl_ResoultionFailure()
         public void Updater_CheckDownloadUrl_ConnectionFailure()
         {
             // this URL produces a connection failure
-            CheckInvalidUrlFails("http://localhost:9000");
+            CheckInvalidUrlFails("http://nonexistent.localhost:9000");
```

Comment:
Minor test fix-up: changing the url so it is more likely to be invalid (previously the test would fail if you have a default installation of SonarQube on the machine).